pkg/kkrtpsi/kkrtpsi.go

```diff
@@ -0,0 +1,135 @@
+package kkrtpsi
+
+import (
+	"encoding/binary"
+	"io"
+	"math"
+	"runtime"
+	"sync"
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/optable/match/internal/cuckoo"
+	"github.com/optable/match/internal/hash"
+	"github.com/optable/match/internal/oprf"
+)
+
+const batchSize = 2048
+
+// HashRead reads one hash
+func EncodesRead(r io.Reader, u *[cuckoo.Nhash]uint64) (err error) {
+	err = binary.Read(r, binary.BigEndian, u)
```

Comment:
Because BigEndian is used in networking protocols. Currently BigEndian is also used in npsi and dhpsi as well. Do you want me to convert them all to LittleEndian?